d2game/d2player/escape_menu.go

```diff
@@ -351,20 +351,15 @@ func (m *EscapeMenu) addEnumLabel(l *layout, optID optionID, text string, values
 	l.actionableElements = append(l.actionableElements, label)
 }
 
-func (m *EscapeMenu) onLoad() {
+func (m *EscapeMenu) OnLoad() {
 	var err error
 	m.selectSound, err = m.audioProvider.LoadSound(d2resource.SFXCursorSelect, false, false)
 	if err != nil {
 		log.Print(err)
 	}
 }
 
-func (m *EscapeMenu) onEscKey() {
-	if !m.isOpen {
-		m.open()
-		return
-	}
```

Comment:
Removed this behavior: we only want the Escape Menu to open if other panels were not closed by this ESC press.